packages/client/test/cli/cli.spec.ts

```diff
@@ -74,7 +70,7 @@ describe('[CLI]', () => {
         // if http endpoint startup message detected, call http endpoint with RPC method
         await wait(600)
         const client = Client.http({
-          port: 8562,
+          port: 8569,
```

Comment:
Out of interest why does the port have to change?

Reply:
I think I had a port collision where multiple child processes (i.e individual tests) were started by `vitest` simultaneously and they both tried to grab the same port.